crates/bevy_app/src/app.rs

```diff
@@ -606,23 +622,31 @@ impl App {
     ///
     /// ```
     /// # use bevy_app::prelude::*;
+    /// # use bevy_app::AppThreading;
     /// #
-    /// let app = App::empty().add_default_stages();
+    /// let app = App::empty().add_default_stages(AppThreading::Multi);
     /// ```
-    pub fn add_default_stages(&mut self) -> &mut Self {
-        self.add_stage(CoreStage::First, SystemStage::parallel())
+    pub fn add_default_stages(&mut self, threading: AppThreading) -> &mut Self {
+        self.add_stage(CoreStage::First, SystemStage::single_threaded())
```

Comment:
Why this was changed from `parallel` to `single_threaded`?

Reply:
nice catch